src/manager.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react'
 import addons from '@storybook/addons'
 import RTLPanel from './containers/RTLPanel'
-import {ADDON_ID, PANEL_ID} from './constants'
+import { ADDON_ID, PANEL_ID } from './constants'
 
 export function register () {
   addons.register(ADDON_ID, (api) => {
     const channel = addons.getChannel()
     addons.addPanel(PANEL_ID, {
       title: 'RTL',
-      render: () => <RTLPanel channel={channel} api={api} />
+      render: props => {
+        /* eslint-disable-next-line react/prop-types */
+        const active = !props || props.active
+        return active ? <RTLPanel channel={channel} api={api} /> : null
```

Comment:
Outside of this line, I pulled in the other PR exactly as it is (and I'm fine to match it exactly/rebase if you choose to merge the other PR).